### PR TITLE
fixed typos in Module Tutorial and adjusted model

### PR DESCRIPTION
**Pull request by Arminta-Jenkins-NIST**
# Committer Notes

{Please provide a brief description of what this PR accomplishes. Be sure to reference any issues addressed. If the PR is a work-in-progress submitted for early review, please include [WIP] at the beginning of the title or mark the PR as DRAFT.}

### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/metaschema/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/metaschema/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [ ] Have you updated all website](https://pages.nist.gov/metaschema) and readme documentation affected by the changes you made? Changes to the website can be made in the website/content directory of your branch.

### CODE REVIEW

**Review by aj-stein-nist — DISMISSED** [review]
I forgot to approve this. I formally approve, since you worked on this while we paired I should make it legitimate now.

File: website/content/tutorials/1-getting-started/_index.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ With an `assembly`, we can specify a complex named object, not just a simple key
             <formal-name>Motherboard Type</formal-name>
             <description>The type motherboard layout, <code>at</code>, <code>atx</code>, <code>mini-itx</code> or an alternative.</description>
           </define-field>
-          <define-assembly name="cpu">
+          <define-assembly name="cpu"min-occurs="1">
```
Comment (iMichaela) [inline website/content/tutorials/1-getting-started/_index.md:242]: <define-assembly name="cpu"min-occurs="1"> space missing before min-occurs

Comment (aj-stein-nist) [reply website/content/tutorials/1-getting-started/_index.md:242]: Good catch.

```suggestion
          <define-assembly name="cpu" min-occurs="1">
```

**Review by aj-stein-nist — APPROVED** [review]

**Review by david-waltermire — APPROVED** [review]
Thanks!

**Pull request merged**